### DISCUSSION TITLE
Problem Adobe Air Windows ARM Mac

### DISCUSSION THREAD

**Discussion by Thomas59290** [airsdk/Adobe-Runtime-Support#3627] [General]
Hi, 

I have a problem since 7 dayss. 
My professiional software that use adobe air get blocked when launching.
I tried to uninstall the app but i have an error message.
![Capture d’écran 2024-12-29 à 18 57 40](https://github.com/user-attachments/assets/30774f39-f98f-4018-bb55-fca544f01d3f)

I can't uninstall it and get blocked.

Can somebody help me ?

Thanks.

Thomas

**Comment by ajwfrost** [comment]
This looks like a duplicate of #3530 and is something we're seeing increasingly, something appears to have changed in Windows to make this start triggering.

We will be uploading a new runtime release shortly that hopefully resolves the installation problem here (i.e. an installer with AIR runtime version 51.1.3.2 in it).

**Reply by ajwfrost** [reply]
@Thomas59290 can you please try with the newly release 51.1.3.4 version to see if that helps?
https://airsdk.harman.com/runtime
thanks

**Comment by Thomas59290** [comment]
Hello, 
i found the solution. 
Apple sequoia blocks a software not signed by hte developper.
Disabling apple security allows this software to work well.

**Reply by ajwfrost** [reply]
So this is curious, aren't you using a virtual machine in which Windows is then running? The installation of an application in Windows shouldn't then be impacted by any security settings from the host operating system, at least as far as I was aware! Maybe they're being really clever about how that works..  

In terms of code signatures .. the AIR runtime software (at least the Windows binaries) should all be signed by Harman with EV certificates. Maybe there's an exception in the template executable that's used to generate new/installed AIR applications (which we modify programmatically to add the icon and file information settings, hence any signing would become invalid).

Anyway .. glad you've got it working now!

thanks
